packages/driver/src/cy/snapshots.ts

```diff
@@ -212,11 +248,38 @@ export const create = ($$, state) => {
 
       const snapshot = {
         name,
-        htmlAttrs,
+        htmlAttrs: $htmlAttrs,
```

Comment:
just curious, do you know why we have pre-fixed these with `$`s?

Reply:
I am guessing some of these are prefixed with `$` because they represent jQuery objects or they use jQuery to calculate their output? TBH I have no idea. I'm also trying to remember the class/instance vs global import discussion as well, but I am guessing this is different since we really aren't importing anything here 😄 .